src/components/Dropdown/Dropdown.scss

```diff
@@ -140,8 +140,7 @@
   cursor: pointer;
   display: block;
   height: 32px;
-  line-height: 30px;
-  padding: 0 32px 0 10px;
+  padding: 5px 32px 0 10px;
```

Comment:
Have you considered using flexbox to align the text, rather than these small adjustments to padding?